api/v1alpha1/kubernetesauthenginerole_types.go

```diff
@@ -127,7 +127,6 @@ func (r *KubernetesAuthEngineRole) IsValid() (bool, error) {
 type VRole struct {
 
 	// TargetServiceAccounts is a list of service account names that will receive this role
-	// +kubebuilder:validation:Required
```

Comment:
why did you remove this?

Reply:
To be able to use default value, removed default instead

Reply:
When I'm nit providing targetServiceAccounts and both required true with default={"default"}, I'm getting:
`error: error validating "./test/kube-auth-engine-role.yaml": error validating data: ValidationError(KubernetesAuthEngineRole.spec): missing required field "targetServiceAccounts" in io.redhat.redhatcop.v1alpha1.KubernetesAuthEngineRole.spec; if you choose to ignore these errors, turn validation off with --validate=false`
`error: error validating "./test/rabbitmq-engine-admin-role.yaml": error validating data: ValidationError(KubernetesAuthEngineRole.spec): missing required field "targetServiceAccounts" in io.redhat.redhatcop.v1alpha1.KubernetesAuthEngineRole.spec; if you choose to ignore these errors, turn validation off with --validate=false`
So, I'm removing this field as default to prevent further confusion.
Changed line number to clearly show that default is removed.

Reply:
let's reinstate the default